### ISSUE TITLE
Unable to download CBeebies programmes

### ISSUE THREAD

**Issue by disconnect78** [Ascoware/get-iplayer-automator#458]
*What steps will reproduce the problem? Be specific, and provide as much detail as possible.*

Attempting to download any programme where the network is CBeebies will immediately fail with the status `Download Failed: DRM protected`. This happens regardless of whether it was added by search, Series-Link, or Use Current Webpage.

*What is the expected output? What do you see instead?*

Logs suggest metadata is being requested from the wrong network:

```
Downloading Hey Duggee: Series 4
ITV: Requesting Metadata
DEBUG: Metadata response status code: 200
**** No metadata found
ERROR: Metadata failure -- most likely this means the show was DRM protected and therefore not supported.
```

*What version of GiA are you using? On what version of OS X?*

GiA 1.25.3, MacOS 12.6

*Are you using a web proxy? DNS proxy? VPN? VPS?*

No.

*What program are you attempting to download (full title, series and episode)? **Provide only ONE program as an example.***

[Hey Duggee S04E01, Norries First Day](https://www.bbc.co.uk/iplayer/episode/m001btvz/hey-duggee-series-4-1-norries-first-day?seriesId=m001btw0)

*Please provide any additional information below.*

Rolling back to 1.25.1 fixes the issue.

The issue also occurs with programmes on S4C eg. [Pawb a'i Farn](https://www.bbc.co.uk/iplayer/episode/p0fnz1c1/pawb-ai-farn-rhaglen-thu-01-jun-2023-2130).

**Comment by skovatch** [comment]
Thanks for the report. I can roll out a fix in the next day or so.

**Comment by disconnect78** [comment]
Ah awesome, thanks! 😁

**Comment by skovatch** [comment]
Fixed in 1.25.4
